docs/content/patterns/alz/deploy/Deploy-with-Azure-CLI.md

```diff
@@ -105,7 +111,7 @@ Note that the parameter file shown below has been truncated for brevity, compare
                 },
                 "ALZMonitorResourceGroupTags": {
                     "value": {
-                        "Project": "amba-monitoring"
+                        "_deployed_by_amba": true
```

Comment:
We should keep the original example value for the tag.

Reply:
Done